test/OpenSpaceToolkit/Astrodynamics/Trajectory/State.test.cpp

```diff
@@ -38,6 +38,17 @@ TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_State, Constructor)
         EXPECT_NO_THROW(State state(instant, coordinates, Frame::GCRF(), brokerSPtr));
     }
 
+    {
+        const Instant instant = Instant::DateTime(DateTime(2018, 1, 1, 0, 0, 0), Scale::UTC);
+        VectorXd coordinates(6);
+        coordinates << 1.0, 2.0, 3.0, 4.0, 5.0, 6.0;
+
+        const Array<Shared<const CoordinatesSubset>> subsets = {
+            CartesianPosition::Default(), CartesianVelocity::Default()
+        };
+        EXPECT_NO_THROW(State state(instant, coordinates, Frame::GCRF(), subsets));
```

Comment:
probably want to check that position is indeed `{1,2,3}` and that velocity is indeed `{4,5,6}`